include/bus_monitor.hpp

```diff
@@ -273,19 +273,21 @@ class SystemStatus
     void rebootPolicyStateCallback(sdbusplus::message::message& msg);
 
     /**
-     * @brief Api to initialize system operating parameters.
+     * @brief Api to initialize system operating mode.
+     *
+     * The API sets the state managers system state based on the operating mode
+     * at the tim eof panel coming up.
      */
-    void initSystemOperatingParameters();
+    void initSystemOpeartingMode();
 
     /**
-     * @brief Set system operating mode.
-     * An api to set system operating mode based on the value of three
-     * parameters required to detect operating mode.
-     * a) Logging Setting.
-     * b) Power policy.
-     * c) Reboot policy.
+     * @brief BIOS attribute change callback.
+     *
+     * Listens to sytem operating mode property change in BIOS base table.
```

Comment:
can we mention here exact attribute name which we are listening  , like sytem_operating_mode property

Reply:
Why? Internal names can be anything, the function is looking for 'a' property which denotes "system operating mode"